types/react-native/test/globals.tsx

```diff
@@ -1,108 +1,150 @@
-const noop = () => { };
+const noop = () => {};
 
 function testInterval() {
     let handle = setInterval(noop, 0);
     clearInterval(handle);
 
-    handle = setInterval((arg1: number, arg2: string) => {
-        console.log('arg1', arg1);
-        console.log('arg2', arg2);
-    }, 0, 100, '200');
+    handle = setInterval(
+        (arg1: number, arg2: string) => {
+            console.log("arg1", arg1);
+            console.log("arg2", arg2);
+        },
+        0,
+        100,
+        "200",
+    );
     clearInterval(handle);
 
-    handle = setInterval((arg1: number, arg2: string) => {
-        console.log('arg1', arg1);
-        console.log('arg2', arg2);
-    // @ts-expect-error
-    }, 0, 'wrong-type', '200');
+    handle = setInterval(
+        (arg1: number, arg2: string) => {
+            console.log("arg1", arg1);
+            console.log("arg2", arg2);
+            // @ts-expect-error
+        },
+        0,
+        "wrong-type",
+        "200",
+    );
     clearInterval(handle);
 
     // @ts-expect-error
     handle = setInterval((missingArg: any) => {
-        console.log('missingArg', missingArg);
+        console.log("missingArg", missingArg);
     }, 0);
     clearInterval(handle);
 
-    handle = setInterval((arg1: number) => {
-        console.log('arg1', arg1);
-    // @ts-expect-error
-    }, 0, 100, 'missing-arg');
+    handle = setInterval(
+        (arg1: number) => {
+            console.log("arg1", arg1);
+            // @ts-expect-error
+        },
+        0,
+        100,
+        "missing-arg",
+    );
```

Comment:
At first glance this looked like a bug in dprint... but I don't see a better way than how dprint is going. Will touch up manually.